package.json

```
@@ -112,6 +112,7 @@
   },
   "devDependencies": {
     "@cspell/dict-es-es": "^3.0.0",
+    "@cspell/dict-fr-fr": "^3.0.0",
```

```suggestion
    "@cspell/dict-fr-fr": "^2.2.2",
```

See https://www.npmjs.com/package/@cspell/dict-fr-fr